solutions/devsprint-michael-douglas-2/DeliveryAppTests/Components/RestaurantInfoViewSnapshotTests.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  RestaurantInfoViewSnapshotTests.swift
+//  DeliveryAppTests
+//
+//  Created by pedro tres on 07/03/22.
+//
+
+import XCTest
+import SnapshotTesting
+@testable import DeliveryApp
+
+class RestaurantInfoViewSnapshotTests: XCTestCase {
+    
+    let size = CGSize(width: 400, height: 100)
+    let isRecord = true
+    
+    func test_restaurantInfoView_whenSnapshot_shouldSucceed() throws {
```

Comment:
Você só precisa utilizar `throws` quando o seu Teste lida com algum tipo de Error Handling, como por exemplo, você fazer o decode de alguma informação e precisar utilizar o `try`. Ai utilizamos o `throws` no método para não precisar fazer o handle do erro com `do catch`.
```suggestion
    func test_restaurantInfoView_whenSnapshot_shouldSucceed() {
```

Reply:
Blza